spec/phlex/component_spec.rb

```diff
@@ -178,6 +178,26 @@ def template
     end
   end
 
+  describe "with underscore in class name" do
+    let :component do
+      Class.new Phlex::Component do
+        def template
+          div class: "a_b"
+        end
+      end
+    end
+
+    it "when convert_underscores_to_dashes config is true convert underscores to dashes and produces the classes" do
+      expect(output).to eq %{<div class="a-b"></div>}
+    end
```

Comment:
Can we add and extra `describe` block to these examples so it doesn't read "it when"?

```suggestion
    describe "when configured to convert_underscores_to_dashes"
      it "converts underscores to dashes and produces the classes" do
        expect(output).to eq %{<div class="a-b"></div>}
      end
    end
```

Reply:
`context` would be better instead of `describe`. Isn't it?